contracts/RMRK/base/RMRKBaseStorage.sol

```diff
@@ -30,19 +30,12 @@ contract RMRKBaseStorage is IRMRKBaseStorage {
 
     uint64[] private _partIds;
 
-    string private _symbol;
+    string private _metadataURI;
     string private _type;
 
-    //TODO: Move to interface
-    //TODO: Doc this struct, put JSON intake format in comments here
-    struct IntakeStruct {
-        uint64 partId;
-        Part part;
-    }
-
-    constructor(string memory symbol_, string memory type__) {
-        _symbol = symbol_;
-        _type = type__;
+    constructor(string memory metadataURI, string memory type_) {
```

Comment:
Is there a reason that `metadataURI` doesn't have a trailing underscore like most of the input arguments?
Also what do you think about changing the capitalisation to `metadataUri`, like we do for `resourceId`?
Both of the changes kind of unify the way we do it in most of the functions.

Reply:
We only add the trailing underscore if needed (to avoid shadowing a method usually). Here there's no need because the method is named "get.."  As for the capitalization I'm indifferent but on ERC721 Metadata extension they use `tokenURI` so uppercase URI feels more standard.

Reply:
Ok, I'm cool with this, people interacting our contracts won't need to worry about it and as long as we know why we do it the way we do, it's fine 👌